provision/acc_provision/test_main.py

```diff
@@ -1771,10 +1938,18 @@ def run_provision(inpfile, expectedkube=None, expectedtar=None,
         copy_file(expectedapic, apicfile, args.debug, "/tmp/generated_apic.txt")
         copy_file(expectedtar, out_tar, args.debug, "/tmp/generated_operator.tar.gz")
 
-        compare_yaml(expectedkube, output, args.debug, "/tmp/generated_kube.yaml", cleanupFunc)
-        compare_yaml(expectedoperatorcr, operator_cr_output, args.debug, "/tmp/generated_operator_cr.yaml", cleanupFunc)
+        if args.compare_kube_yaml_aci_op_cm_in_plain_text:
+            result_kube_yaml = compare_kube_yaml(expectedkube, output, args.debug, "/tmp/generated_kube.yaml", cleanupFunc)
+            result_op_cr = compare_kube_yaml(expectedoperatorcr, operator_cr_output, args.debug, "/tmp/generated_operator_cr.yaml", cleanupFunc)
+            compare_tar_plaintext(expectedtar, out_tar.name, args.debug, "/tmp/generated_operator.tar.gz", cleanupFunc)
+            assert result_kube_yaml is True, cleanupFunc()
+            assert result_op_cr is True, cleanupFunc()
+        else:
```

Comment:
do we need the else part? if the default way of checking will be using base decode/encode